libs/blocks/share/share.js

```diff
@@ -37,35 +37,58 @@ export async function getSVGsfromFile(path, selectors) {
 
 function getPlatforms(el) {
   const manualShares = el.querySelectorAll('a');
-  if (manualShares.length === 0) return null;
-  return [...manualShares].map((link) => {
-    const { href } = link;
+  if (manualShares.length === 0) return ['facebook', 'twitter', 'linkedin', 'pinterest', 'reddit'];
+  const platforms = [];
+  [...manualShares].forEach((share) => {
+    const { href } = share;
     const url = new URL(href);
     const parts = url.host.split('.');
-    return parts[parts.length - 2];
+    platforms.push(parts[parts.length - 2]);
+    const parentP = share.closest('p');
+    parentP?.remove();
   });
+  return platforms;
 }
 
 export default async function decorate(block) {
   const config = getConfig();
   const base = config.miloLibs || config.codeRoot;
-  const platforms = getPlatforms(block) || [
-    'facebook',
-    'twitter',
-    'linkedin',
-    'pinterest',
-    'reddit',
-  ];
-  block.innerHTML = '';
+  const platforms = getPlatforms(block);
+  const rows = block.querySelectorAll(':scope > div');
+  const childDiv = rows[0].querySelector(':scope > div');
+  const emptyRow = childDiv?.innerText.trim() === '';
+  /* eslint-disable  no-confusing-arrow,no-useless-escape */
```

Comment:
nit: I know there's a pre-existing comment like this on this file, but we should really only be disabling eslint rules for specific lines/blocks of code. As it is right now, this comment would disable those rules for the entire file... Probably best practice to use [`eslint-disable-next-line` or add an `eslint-enable`](https://eslint.org/docs/latest/use/configure/rules#disabling-rules) at the end of the block you're targeting.